### ISSUE TITLE
[data request] <Cityscapes>

### ISSUE THREAD

**Issue by zishanahmed08** [tensorflow/datasets#214]
* Name of dataset: Cityscapes
* URL of dataset: https://www.cityscapes-dataset.com/
* License of dataset: This dataset is made freely available to academic and non-academic entities for non-commercial purposes such as academic research, teaching, scientific publications, or personal experimentation. Permission is granted to use the data given that you agree:

That the dataset comes “AS IS”, without express or implied warranty. Although every effort has been made to ensure accuracy, we (Daimler AG, MPI Informatics, TU Darmstadt) do not accept any responsibility for errors or omissions.
That you include a reference to the Cityscapes Dataset in any work that makes use of the dataset. For research papers, cite our preferred publication as listed on our website; for other media cite our preferred publication as listed on our website or link to the Cityscapes website.
That you do not distribute this dataset or modified versions. It is permissible to distribute derivative works in as far as they are abstract representations of this dataset (such as models trained on it or additional annotations that do not directly include any of our data) and do not allow to recover the dataset or something similar in character.
That you may not use the dataset or any derivative work for commercial purposes as, for example, licensing or selling the data, or using the data with a purpose to procure a commercial gain.
That all rights not expressly granted to you are reserved by us (Daimler AG, MPI Informatics, TU Darmstadt).
* Short description of dataset and use case(s): We present a new large-scale dataset that contains a diverse set of stereo video sequences recorded in street scenes from 50 different cities, with high quality pixel-level annotations of 5 000 frames in addition to a larger set of 20 000 weakly annotated frames. The dataset is thus an order of magnitude larger than similar previous attempts. Details on annotated classes and examples of our annotations are available at this webpage.

The Cityscapes Dataset is intended for

1. assessing the performance of vision algorithms for two major tasks of semantic urban scene understanding: pixel-level and instance-level semantic labeling;

2. supporting research that aims to exploit large volumes of (weakly) annotated data, e.g. for training deep neural networks.

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.

And if you'd like to contribute the dataset (thank you!), see our [guide to adding a dataset](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).

**Comment by nikste** [comment]
I am also HIGHLY interested in this !

**Comment by f1recracker** [comment]
I've been working on this and will be sending a pull request soon!

**Comment by zishanahmed08** [comment]
@f1recracker Harsh, Is there  a tentative date as to when your PR will be merged?

**Comment by f1recracker** [comment]
Sorry - I've been a bit busy with school and finals but I just sent an updated PR that _should_ fix all previous blockers :)

**Comment by mathmanu** [comment]
This is an important dataset. Thanks for working on it. Can you take care of the changes requested by the reviewer?

**Comment by f1recracker** [comment]
I understand. I've already implemented all changes so far.

**Comment by zishanahmed08** [comment]
@f1recracker @mathmanu any updates?

**Comment by mathmanu** [comment]
Hi @f1recracker, It looks like there is one pending comment on your PR, can you address the comment  https://github.com/tensorflow/datasets/pull/225 and then make sure it is merged?

**Comment by f1recracker** [comment]
Hey @mathmanu. Sorry for the late response. I just rechecked all comments and I have addressed all of them in the past. Some conversations were still marked as unresolved which have now been closed.
